docs/before-you-begin.md

```diff
@@ -1,4 +1,4 @@
-# Before You Begin
+## Before You Begin
```

Comment:
this is h2 instead of h1 because these will be imported from an `overview.md` parent file that will have its own `# overview` title.